src/Pkg.jl

```diff
@@ -130,12 +130,13 @@ See also [`PackageSpec`](@ref).
 const add = API.add
 
 """
-    Pkg.precompile()
+    Pkg.precompile(; throw_indirect_deps::Bool=false)
```

Comment:
Perhaps a simpler keyword like
```suggestion
    Pkg.precompile(; strict::Bool=false)
```
would be nicer?

Reply:
Ok, yeah. That's I think what @omus was suggesting too. I'll go with it